assets/scss/application.scss

```diff
@@ -4,7 +4,12 @@ $path: '/assets/images/';
 $moj-page-width: 1170px;
 $govuk-page-width: $moj-page-width;
 
-@import 'govuk/all';
+@import "node_modules/govuk-frontend/dist/govuk/base";
```

Comment:
Replaced as required in release notes: https://github.com/alphagov/govuk-frontend/releases/tag/v5.4.0